app/src/main/java/it/niedermann/owncloud/notes/edit/title/EditTitleDialogFragment.java

```diff
@@ -56,12 +59,25 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
         return new AlertDialog.Builder(getActivity())
                 .setTitle(R.string.change_note_title)
                 .setView(dialogView)
-                .setCancelable(true)
-                .setPositiveButton(R.string.action_edit_save, (dialog, which) -> listener.onTitleEdited(binding.title.getText().toString()))
-                .setNegativeButton(R.string.simple_cancel, null)
+                .setCancelable(false)
+                .setPositiveButton(R.string.action_edit_save, (dialog, which) -> {
+                    hideKeyboard(dialogView.findViewById(R.id.title).getWindowToken());
+                    listener.onTitleEdited(binding.title.getText().toString());
+                })
+                .setNegativeButton(R.string.simple_cancel, new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialogInterface, int i) {
+                        hideKeyboard(dialogView.findViewById(R.id.title).getWindowToken());
+                    }
+                })
                 .create();
     }
 
+    private void hideKeyboard(IBinder windowToken) {
+        InputMethodManager inputManager = (InputMethodManager) getContext().getSystemService(Context.INPUT_METHOD_SERVICE);
```

Comment:
use `requireContext()` to ensure it is not `null`

Reply:
Please use `final` here for the `inputManager`

Reply:
okay